### PR TITLE
Added template for forms (login, signup, application)

### LINKED ISSUE #125: Add login/sign up template

### PR DESCRIPTION
**Pull request by lisa-sa-li**
Style only, not connected to anything yet. 

Inputs will be removed later and replaced with Django

### CODE REVIEW

File: hackathon_site/event/jinja2/event/login.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{% extends "event/form_base.html" %}
+
+{% block form_title %}Login{% endblock %}
+{% block form_button %}Login{% endblock %}
+
+{% block form_img %}<img src="{{ static('event/images/landing.svg') }}" />{% endblock %}
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/login.html]: img needs an alt attribute (I'm just reading off PyCharm warnings)

File: hackathon_site/event/jinja2/event/form_base.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+<div class="container">
+    <div class="section">
+        <div class="loginDiv z-depth-3 row">
+            <div class="row col s12 flexColCenter {% if self.form_img() %}l7 m8{% endif %}">
+                <h1 class="formH1">{% block form_title %}{% endblock %}</h1>
+                <form class="col s12">
+                    {% block form_inputs %}{% endblock %}
+                </form>
+
+                <a href="#" class="btn-small waves-effect waves-light secondaryColor colorBtn center">{% block form_button %}{% endblock %}</a>
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/form_base.html]: Each form is going to need a way to alter what this button does, probably by making it an actual button with an `onClick`. Having the classes in the base template is nice, but I think the `form_button` block needs to capture everything, and each instance will have to apply the classes. Any workarounds?

File: hackathon_site/event/jinja2/event/login.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{% extends "event/form_base.html" %}
+
+{% block form_title %}Login{% endblock %}
+{% block form_button %}Login{% endblock %}
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/login.html]: A mentioned above, this is where we'd have to include the logic for submitting the form

File: hackathon_site/event/jinja2/event/signup.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
+{% extends "event/form_base.html" %}
+
+{% block form_title %}Sign up{% endblock %}
+{% block form_button %}Sign up{% endblock %}
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/signup.html]: Likewise, there would have to be logic here to handle submitting the form

File: hackathon_site/event/jinja2/event/signup.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+{% block form_img %}<img src="{{ static('event/images/landing.svg') }}" />{% endblock %}
+               
+{% block form_inputs %}
+    <div class="input-field col s12">
+        <input id="first_name" type="text" class="validate">
+        <label for="first_name">first name</label>
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/signup.html:11]: Should all these labels be capitalized (`First Name` or `First name`, etc)? Looks strange to me with them all being in lowercase

File: hackathon_site/event/jinja2/event/signup.html [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+        <input id="email" type="email" class="validate">
+        <label for="email">email</label>
+    </div>
+    <div class="input-field col s12">
+        <input id="password" type="password" class="validate">
+        <label for="password">password</label>
+        <span class="helper-text">Must contain blah blah</span>
+    </div>
+    <div class="input-field col s12">
+        <input id="password" type="password" class="validate">
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/signup.html:27]: Two things with the same id. `password-confirm` or something here (I think the Django form will set the id down the line anyway)

File: hackathon_site/event/jinja2/event/signup.html [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+{% block form_bottom %}
+<div class="col s12 loginLink">
+    <i class="material-icons primaryText">exit_to_app</i>
+    <a href="#" class="primaryText hoverLink s6">Back to Login</a>
+</div>
+<br />
+<div class="row center">
+    Something wrong? <a href="#" class="primaryText hoverLink">Contact us</a>.
+</div>
+{% endblock %}
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/signup.html]: Minor, newline at the end of the file

File: hackathon_site/event/jinja2/event/login.html [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+
+{% block form_img %}<img src="{{ static('event/images/landing.svg') }}" />{% endblock %}
+               
+{% block form_inputs %}
+<div class="row" style="margin-bottom: 0;">
+    <div class="input-field col s12" style="margin-bottom: 0;">
+        <input id="email" type="email" class="validate">
+        <label for="email">email</label>
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/login.html:12]: As above, capitalize labels?

File: hackathon_site/event/jinja2/event/dashboard_base.html [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+
+            <h4 class="formH2 col s12">Complete your application</h4>
+            <p><strong>Status</strong>: incompleted</p>
+            <a class="btn-small waves-effect waves-light secondaryColor colorBtn center">Get started</a>
+        </div>
+
+        <div class="borderTopDiv z-depth-3">
+            <h4 class="formH2 col s12">Apply as team</h4>
+
+            <p>Have friends you want to work with? Create a team with up to 4 people and we’ll review your applications together. Enter your team code or make a new one and share with your members.</p>
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/dashboard_base.html:17]: From an implementation perspective this year, everyone will be on a team by default. They can join another team and automatically leave theirs, and when they leave another team they'll be automatically put in a new team of 1.

File: hackathon_site/event/jinja2/event/application.html [hunk 10]
```diff
@@ -0,0 +1,110 @@
+{% extends "event/form_base.html" %}
+
+{% block form_title %}Application{% endblock %}
+{% block form_button %}Submit{% endblock %}
+               
+{% block form_inputs %}
+    <h4 class="formH2 col s12">Personal</h4>
+    <div class="input-field col m7 s12">
+        <input id="birthday" type="text" class="validate">
+        <label for="birthday">birthday</label>
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/application.html:10]: Lowercase labels still spook me

File: hackathon_site/event/jinja2/event/application.html [hunk 11]
```diff
@@ -0,0 +1,110 @@
+{% extends "event/form_base.html" %}
+
+{% block form_title %}Application{% endblock %}
+{% block form_button %}Submit{% endblock %}
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/application.html:4]: Once again, action will have to go in here

File: hackathon_site/event/jinja2/event/application.html (hunk 10 above)
Comment (lisa-sa-li) [reply hackathon_site/event/jinja2/event/application.html:10]: lol these dont matter cause it'll be Django inputa later right

File: hackathon_site/event/jinja2/event/dashboard_base.html (hunk 9 above)
Comment (lisa-sa-li) [reply hackathon_site/event/jinja2/event/dashboard_base.html:17]: So will they have a code displayed already? I'll fix this dashboard page in a future PR. Just wanted the shell already done

Comment (grahamhoyes) [reply hackathon_site/event/jinja2/event/dashboard_base.html:17]: Yes, they'll already have a team code

**Review by grahamhoyes — APPROVED** [review]
LGTM. The password ID is still duplicated on the sign up form, but that will be fixed when the django form generates the fields

**Pull request merged**